ServiceBoxWeb/src/main/java/com/infoshareacademy/controller/employee/ServiceOrderController.java

```diff
@@ -65,4 +64,24 @@ public String changeServiceOrderState(@PathVariable("id") Long serviceOrderId) {
         serviceOrderService.updateStatus(serviceOrderId);
         return "redirect:/employee/service-orders";
     }
+
+    @GetMapping("/create")
```

Comment:
nie uzywamy sugestywnych nazw endpointow 
https://nordicapis.com/10-best-practices-for-naming-api-endpoints/